content/articles/sending-sms-twilio-in-flutter/index.md

```diff
@@ -0,0 +1,228 @@
+```
+Next, we will define the constructor that will receive the SMS received text, which is saved to a string.
+
+Then using that string we will send an update to all the current messages in our database.
+
+### Step3 . Sending SMS with update list data:
+
+```dart
+List<String> _messageList = new List(); // Add Twilio dependency for sending sms messages.
+List<String> messagesToSend = _messageList.map((x) => new
+String msg = x.replace(" ", "").toLowerCase(); // Add Twilio dependency for sending sms messages.
+```
+
+Up next, we will define a new method to send SMS with our Database's current list of messages.
+
+### Step4. Sending SMS from the Database:
+
+```dart
+String message = ""; // Add Twilio dependency for sending sms messages.
+_messageList.forEach((x) =>
+{ 
+welcomeMsg.post(message);
+ }
+ );
+```
+
+In the previous step, we defined a message that will be sent from our SMS controller. I have called this method to send SMS. Now let's proceed with sending an update to all the list data from our database. This is after typing a new text and pressing submit button.
+```dart
+String msg = x.replace(" ", "").toLowerCase(); // Add Twilio dependency for sending sms messages.
+message = msg + " : new update from database, now there are "+_messageList.length+" messages"; // Add Twilio dependency for sending sms messages.
+```
+After typing a new SMS text and pressing the button. Our controller will receive the text and send an update to all messages in our database.
+The last step is to process the SMS received method. In this case, we will use that string message to save it in our database. Then send a reply back with a confirmation message. Confirmation message states if message was sent  successfully or a failure. It's also a notification if the data could not be saved.
+
+### Step5. Processing SMS received method:
+
+```dart
+String msg = x.replace(" ", "").toLowerCase(); // Add Twilio dependency for sending sms messages.
+message = msg + " : new update from database, now there are "+_messageList.length+" messages"; // Add Twilio dependency for sending sms messages.
+_messageList.forEach((x) =>
+{
+ welcomeMsg.post(message);
+  }
+);
+
+```
+### Step6. Sending messages to user with Twilio:
+
+Last step, we will actually send the messages to the user that is sending SMS to our Twilio number.
+Finally, we will clear the input field on send button click. Then show some feedback message if the reply was successful or failure notification.
+Sending SMS with update list data
+` List<String> _mes` can send SMS with current list data from the Database to any number using the Twilio service. We will put one more screen just to show the final look of our application before testing it.
+
+```dart
+
+sageList = new List(); // Add Twilio dependency for sending sms messages.
+List<String> messagesToSend =String msg = x.replace(" ", "").toLowerCase(); // Add Twilio dependency for sending sms messages.
+```
+### Conclusion
```

Comment:
Include any next or final steps summarizing the tasks the audience just completed with possible next steps.